MOTIVATION.md

```diff
@@ -0,0 +1,138 @@
+# Motivation
+
+The Human Cell Atlas (HCA) Data Coordination Platform (DCP) ingests,
+processes, and stores single-cell data with the goal of making that data easily
+available to a wide audience of researchers.
+The single-cell data type of greatest interest to most researchers is expression
+matrices, matrices that contain cell-by-gene expression values that are the
+starting point for many downstream analyses.
+The importance of this data type calls for a specialized interface that
+maximizes its utility. This specialized interface is called the DCP "matrix
+service". The matrix service is responsible for enabling investigators to access
+and analyze HCA expression data as easily as possible.
+
```

Comment:
I'm not keen on the repetition of _specialized interface_. Building on Ambrose's suggestion, how about:

``To support this most commonly requested data type, the DCP offers a specialized interface, the DCP matrix service, which enables investigators to easily access and analyze HCA expression data.``

Reply:
:heart: